pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -703,6 +688,55 @@ func newKibanaDeployment(namespace string) *v1beta2.Deployment {
 	return deployment
 }
 
+func UpdateEmbeddedEndpointWithRetry(podLister rv1.PodLister, serviceLister rv1.ServiceLister, clusterLoggings v3.ClusterLoggingInterface, nodeLister v3.NodeLister, clusterName string, logger logstream.LoggerStream) error {
+	timeout := time.After(2 * time.Minute)
+	ticker := time.NewTicker(10 * time.Second)
+	done := make(chan bool, 1)
+
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	updated := false
+	errCh := make(chan error, 12)
+	go func() {
+		defer wg.Done()
+		for {
+			select {
+			case <-ticker.C:
+				err := UpdateEmbeddedEndpoint(podLister, serviceLister, clusterLoggings, nodeLister, clusterName)
+				if err != nil {
+					logger.Infof("get embedded components status failed, %s", err.Error())
```

Comment:
Can we show the progress of the deployment of the embedded target? like it is pulling image